src/plugins/vis_type_timeseries/public/application/visualizations/views/timeseries/index.js

```diff
@@ -54,14 +56,15 @@ export const TimeSeries = ({
   legend,
   legendPosition,
   tooltipMode,
-  xAxisLabel,
   series,
   yAxis,
   onBrush,
   xAxisFormatter,
   annotations,
   syncColors,
   palettesService,
+  interval,
+  shouldDropLastBucket,
```

Comment:
In fact, the drop bucket happens on the server side. I think the name of the variable does not reflect the essence of what you want. Please rename